verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -593,9 +593,19 @@ void IndexingFactsTreeExtractor::ExtractMethodCallExtension(
   IndexingNodeData function_node_data(IndexingFactType::kFunctionCall);
   IndexingFactNode function_node(function_node_data);
 
+  if (facts_tree_context_.top().Children().size() == 0) {
```

Comment:
Use .empty() instead of .size() == 0

Reply:
Is this a malformed CST? Maybe add a VLOG?

Same below.

Reply:
this crashes if the statement was `super.function_call()` so I added a handling for it and it can be left just to avoid any other case like this one.